src/Shell/Task/ExtractTask.php

```diff
@@ -122,7 +122,8 @@ protected function _getPaths()
             $response = $this->in($message, null, $defaultPath);
             if (strtoupper($response) === 'Q') {
                 $this->err('Extract Aborted');
-                return $this->_stop();
+                $this->_stop();
+                return;
```

Comment:
Phpcs complains about method returning a value in spite of having `@return void`.